### PR TITLE
Update release notes for 0.146

### PR DESCRIPTION
**Pull request by geraint0923**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ General Changes
 ---------------
 
 * Require at least 4096 file descriptors to run Presto.
+* Casting ``MAP`` types is supported.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: This should be something like "Add support for casting between map types."

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: This should be a verb rather than a statement of the current state

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: @martint gave a good way to think about this: If you prefix the release note with "After updating, Presto will ..."

Thus, compare these two:

* After updating, Presto will casting map types is supported.
* After updating, Presto will support casting between map types.

**Comment by geraint0923** [comment]
Address the comment.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**